[#1] 백준 14888 주용

cheesecat47 opened the pull request:
#1

cheesecat47 committed:
[#1] 백준 14888 주용

cheesecat47 commented on 14888. 연산자 끼워넣기/Nojam14888_230724_주용.java:
```diff
@@ -0,0 +1,66 @@
+			else if (opIdx == 2) newResult *= numbers[numIdx];
+			else {
+				if (newResult < 0) {
+					newResult *= -1;
```
이부분 문제에 나온 그대로 했는데 성규 코드 보니 그냥 나누기 해도 되나보던데...

https://github.com/AlgoStudy-Gumi10th-Class4/Algorithm-Study/blob/b6df7eec0c6db60ed055022c2d12c6bc92dfd30b/14888.%20%EC%97%B0%EC%82%B0%EC%9E%90%20%EB%81%BC%EC%9B%8C%EB%84%A3%EA%B8%B0/0724%EC%84%B1%EA%B7%9C.java#L49

Gyu-S replied:
처음 시도 때 음수에서 오류 나서 그게 문제인가 했는데 MAX_VAL, MIN_VAL 초기 값만 잘 설정해주면 문제 없이  돌아가더라고요

cheesecat47 committed:
Merge branch 'AlgoStudy-Gumi10th-Class4:main' into main

cheesecat47 committed:
BOJ 2839 채점 결과 업데이트

cheesecat47 committed:
컨벤션 추가 (#1)

- 폴더 명이나 파일 명 컨벤션은 회의 후 수정 가능 (우선 예시로 추가)
- 커밋 메시지 템플릿은 .gitmessage.txt 파일 아래 참고하여 적용
